ranger/data/scope.sh

```diff
@@ -147,6 +176,12 @@ handle_image() {
         #     ebook-meta --get-cover="${IMAGE_CACHE_PATH}" -- "${FILE_PATH}" > /dev/null \
         #         && exit 6 || exit 1;;
 
+	# ePub, FB2 (using pandoc)
```

Comment:
Oh, wait. This doesn't work. You need to output a jpeg in the handle_image() function I think.

Reply:
So let's just add these formats to the docx clause maybe. We can treat it as the general pandoc clause for now.

Reply:
But if it needs an extra function, it won't work there either, will it? So why not get rid of it for now?

Reply:
I added some comments about not being able to test pandoc with those formats in a thread you marked as resolved. So I don't know if you saw my messages.

Reply:
Oh, I see, I moved it to another function.

Reply:
I moved it quickly and I did not realize that I moved from the `handle_extension()` to the `handle_image()` function :grimacing: 

Will move it back with .docx.

Reply:
I saw, no worry. I trust pandoc implicitly : ) (getting inundated with emails from all your comments so I'm not missing *anything* : ))


What do you mean extra function? It's just adding `|epub|fb2` to the `docx` clause. It'll give nicer epub previews without needing image preview support.

Reply:
I deleted that comment. It was a misunderstanding before I realized my mistake.

I moved it back in the extension fonction. Have to leave now, but will check later if you would like more changes.